scripts/task_process/task_proc_wrapper.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+if [ ! -f /etc/enable_status_daemon ]; then
+    echo "/etc/enable_status_daemon file not found, not starting the status daemon and exiting"
+    exit 1
+fi
+
+TIME_BETWEEN_QUERIES=24 #hours
+# SECONDS variable is always being automatically incremented.
+# for the beginning of this script, set it to a big value so that condor_q can be done initially
+SECONDS=$(($TIME_BETWEEN_QUERIES * 3601))
```

Comment:
I think you can achieve the same thing by setting CONDOR_RES to something (e.g.: 0). Would be cleaner imho